packages/kbn-dev-utils/src/ci_stats_reporter/ci_stats_reporter.ts

```diff
@@ -99,6 +108,14 @@ export class CiStatsReporter {
       cpuSpeed: Os.cpus()[0]?.speed,
       freeMem: Os.freemem(),
       totalMem: Os.totalmem(),
+      memoryUsageRss: memUsage.rss,
+      memoryUsageHeapTotal: memUsage.heapTotal,
+      memoryUsageHeapUsed: memUsage.heapUsed,
+      memoryUsageExternal: memUsage.external,
+      memoryUsageArrayBuffers: memUsage.arrayBuffers,
```

Comment:
Not sure how helpful these are going to be, but I am not opposed to leaving them if you think they will have value.